db/migrate/20200517190138_rename_crop_fields.rb

```diff
@@ -0,0 +1,23 @@
+class RenameCropFields < ActiveRecord::Migration[5.2]
+  def up
```

Comment:
IIRC `rename_column` is a reversible operation; meaning that you can rename `#up` to `#change` and get rid of the `#down` method.

Reply:
Neat!